action.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+use dokuwiki\plugin\twofactor\Provider;
+use dokuwiki\Form\Form;
+
+// Load the PHP QR Code library.
+require_once(dirname(__FILE__).'/phpqrcode.php');
```

Comment:
any chance we can get a better library? in any case this should be autoloaded via namespace